libctru/source/applets/miiselector.c

```diff
@@ -0,0 +1,42 @@
+#include <3ds/types.h>
+#include <3ds/services/apt.h>
+
+#include <3ds/applets/miiselector.h>
+
+Result miiSelectorLaunch(MiiSelectorContext *ctx)
```

Comment:
From above, this would look like the following:

```
Result miiSelectorLaunch(const MiiSelectorConf *conf, MiiSelectorReturn *returnbuf);
{
	union {
		MiiSelectorConf config;
		MiiSelectorReturn ret;
	} ctx;

	memcpy(&ctx.config, conf, sizeof(MiiSelectorConf));
	ctx.config.magic = MIISELECTOR_MAGIC;

	Result ret = aptLaunchLibraryApplet(APPID_APPLETED, &ctx->config, sizeof(MiiSelectorConf), 0);
	if(R_SUCCEEDED(ret) && returnbuf)
		memcpy(returnbuf, &ctx.ret, sizeof(MiiSelectorReturn));

	return ret;
}
```